internal/hud/webview/convert_test.go

```diff
@@ -205,8 +206,110 @@ func TestLocalResource(t *testing.T) {
 	r := v.Resources[1]
 	assert.Equal(t, "test", r.Name)
 	assert.Equal(t, model.RuntimeStatusNotApplicable, model.RuntimeStatus(r.RuntimeStatus))
+	require.Len(t, r.Specs, 1)
+	spec := r.Specs[0]
+	require.Equal(t, proto_webview.TargetType_TT_LOCAL, spec.Type)
+	require.False(t, spec.HasLiveUpdate)
 }
 
+func TestBuildHistory(t *testing.T) {
```

Comment:
wanted test coverage for my change, but unsure if this is too finicky, thoughts?

Reply:
this is fine, tho it might be worthwhile to eventually invest in a kind of golden test around this